index.html

```diff
@@ -0,0 +1,325 @@
+        <div class="blog__paragraf">
+          Dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+          Duis aute irure dolor in reprehenderit in voluptate velit.
+        </div>
+        <div class="blog__list">
+          <ul class="blog__wrapper">
+            <li class="blog__item product product__bg_apple">
+              <div class="product__title">
+                Green Smoothies: Too Much of a Good Thing?
+              </div>
+              <div class="product__subtitle">By Auskteez - 2 hours ago</div>
+              <div class="product__number"><span>20</span> nov</div>
+              <div class="product__gradient">
+                <img
+                  src="images/Gradient.png"
+                  alt="gradient"
+                  class="product__img"
+                />
+              </div>
+            </li>
+            <li class="blog__item product product__bg_green">
+              <div class="product__title">
+                Green Smoothies: Too Much of a Good Thing?
+              </div>
+              <div class="product__subtitle">By Auskteez - 2 hours ago</div>
+              <div class="product__number"><span>14</span> nov</div>
+              <div class="product__gradient">
+                <img
+                  src="images/Gradient.png"
+                  alt="gradient"
+                  class="product__img"
+                />
+              </div>
+            </li>
+            <li class="blog__item product product__bg_malina">
+              <div class="product__title">
+                Green Smoothies: Too Much of a Good Thing?
+              </div>
+              <div class="product__subtitle">By Auskteez - 2 hours ago</div>
+              <div class="product__number"><span>10</span> nov</div>
+              <div class="product__gradient">
+                <img
+                  src="images/Gradient.png"
+                  alt="gradient"
+                  class="product__img"
+                />
+              </div>
+            </li>
+          </ul>
+        </div>
+        <button class="btn product__btn" type="button">View More</button>
+      <!-- </div> -->
+    </section>
+    <section name="contact" class="contact">
+      <div class="container">
+        <h2 class="h2-title contact__title">Contact</h2>
+        <div class="contact__subtitle">
+          For new projects, ideas, business or anything you'd like to discuss,
+          contact with me!
+        </div>
+
+        <div class="contact__wrapper">
+          <form action="" class="contact__form">
+            <input id="text" class="contact__input" type="text" /
+            placeholder="Auskteez Tran"> <input id="email"
+            class="contact__input" type="email" / placeholder="Your email...">
+            <input id="tel" class="contact__input" type="tel" /
+            placeholder="Your phone number...">
```

Comment:
тут шось не так, теж видно опечатка
```suggestion
            <input
              id="text"
              class="contact__input"
              type="text"
              placeholder="Auskteez Tran"
            />
            <input
              id="email"
              class="contact__input"
              type="email"
              placeholder="Your email..."
            />
            <input
              id="tel"
              class="contact__input"
              type="tel"
              placeholder="Your phone number..."
            />
```